monai/networks/blocks/segresnet_block.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Type, Union
+
+import torch.nn as nn
+
+from monai.networks.layers.factories import Act, Conv, Norm
+
+
+def get_norm_layer(spatial_dims: int, in_channels: int, norm_name: str, num_groups: int = 8):
+    if norm_name not in ["batch", "instance", "group"]:
+        raise ValueError(f"Unsupported normalization mode: {norm_name}")
+    else:
+        if norm_name == "group":
+            norm = Norm[norm_name](num_groups=num_groups, num_channels=in_channels)
+        else:
+            norm = Norm[norm_name, spatial_dims](in_channels)
+        if norm.bias is not None:
+            nn.init.zeros_(norm.bias)
+        if norm.weight is not None:
+            nn.init.ones_(norm.weight)
+        return norm
+
+
+def get_conv_layer(
+    spatial_dims: int, in_channels: int, out_channels: int, kernel_size: int = 3, stride: int = 1, bias: bool = False
+):
+
+    conv_type: Type[Union[nn.Conv1d, nn.Conv2d, nn.Conv3d]] = Conv[Conv.CONV, spatial_dims]
+
+    conv = conv_type(
```

Comment:
could line 38-42 be implemented with `monai.networks.blocks.Convolution`?

Reply:
Thanks! I changed it.